fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1153,21 +1129,23 @@ def _init_param_attributes(self, p: Parameter) -> None:
             return
 
         # A single shard of the parameters in full precision.
+        # TODO: PJ - I believe this will cause memory leakage with ssd
```

Comment:
The convention for TODOs are using "TODO(username): ...." Can we reformat this to follow that convention? might be good to also add context on why you think this will cause a memory leak.

Shall we add this memory leak investigation to our task tracker as well? (maybe you already did but just checking)